Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Staging/SetPoints/Subsequences/Validation/BoilerIndices.mo

```diff
@@ -0,0 +1,143 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.BoilerPlant.Staging.SetPoints.Subsequences.Validation;
+model BoilerIndices
+  "Validates extraction of boiler indices in a given stage"
+
+  Buildings.Controls.OBC.ASHRAE.PrimarySystem.BoilerPlant.Staging.SetPoints.Subsequences.BoilerIndices
```

Comment:
Michael decided to not mandate the location of the tested instances, but still recommend they be grouped together.